development/skinning.md

```diff
@@ -0,0 +1,182 @@
+  Panel (Desktop) |  | 
+ `--alpheios-desktop-panel-header-bg` | `--alpheios-toolbar-bg-color` | Panel header background color
+   | | 
+ `--alpheios-desktop-panel-icon-color` | `--alpheios-icon-color` | Panel header icons color
+ `--alpheios-desktop-panel-icon-bg` | `--alpheios-toolbar-bg-color` |  Panel header icons background color
+ `--alpheios-desktop-panel-icon-color-hover` | `--alpheios-icon-color-hover` |  Panel header icons color - hover state
+ `--alpheios-desktop-panel-icon-color-active` | `--alpheios-icon-color-active` |  Panel header icons color - active state
+ `--alpheios-desktop-panel-icon-bg-active` | `--alpheios-toolbar-bg-color` |  Panel header icons background color - active state
+ `--alpheios-desktop-panel-icon-color-disabled` | `--alpheios-icon-color-disabled` |   Panel header icons background color - disabled state
+ Panel (Compact) |  | 
+ `--alpheios-compact-panel-header-bg` | `--alpheios-toolbar-bg-color` | Panel header background color
+ `--alpheios-compact-panel-content-bg` | `--alpheios-text-bg-color` | Panel content background color
+ | | 
+ `--alpheios-compact-panel-icon-color` | `--alpheios-icon-color` | Panel header icons color
+ `--alpheios-compact-panel-icon-bg` | `--alpheios-toolbar-bg-color` |  Panel header icons background color
+ `--alpheios-compact-panel-icon-color-hover` | `--alpheios-icon-color-hover` | Panel header icons color - hover state
+ `--alpheios-compact-panel-icon-bg-hover` | `--alpheios-toolbar-bg-color` | Panel header icons background color - hover state
+ `--alpheios-compact-panel-icon-color-active` | `--alpheios-icon-color-active` | Panel header icons color - active state
+ Lookup |  | 
+ `--alpheios-lookup-border-color` | `--alpheios-border-color` | Lookup panel border color (in toolbar)
+ `--alpheios-lookup-bg` |  `--alpheios-text-bg-color` | Lookup panel panel background (in toolbar)
+ `--alpheios-lookup-input-border-color` | `--alpheios-border-color` | Lookup input border color
+ `--alpheios-lookup-button-color` | `--alpheios-btn-primary-font-color` | Lookup main button - font color
+ `--alpheios-lookup-button-bg` | `--alpheios-btn-primary-bg-color` | Lookup main button - background color
+ `--alpheios-lookup-button-border-color` | `--alpheios-btn-primary-border-color` | Lookup main button - border color
+ `--alpheios-lookup-button-color-hover` | `--alpheios-btn-primary-font-color-hover` | Lookup main button - font color - hover state
+ `--alpheios-lookup-button-bg-hover` | `--alpheios-btn-primary-bg-color-hover` | Lookup main button - background color - hover state
+ `--alpheios-lookup-button-border-color-hover` | `--alpheios-btn-primary-border-color-hover` | Lookup main button - border color - hover state
+ Inflections Browser |  | 
+ `--alpheios-inflect-browser-title-color` | `--alpheios-link-color-on-light` | Inflections Browser - main title - font color

```

Comment:
All following items starts with `--alpheios-inflect` prefix before the text describing the location. This one starts with `--alpheios-inflect-browsers`. Maybe it's better to use `--alpheios-inflect` for consistency?

Reply:
agree

Reply:
fixed